compiler/src/dotty/tools/dotc/ast/Trees.scala

```diff
@@ -677,23 +677,41 @@ object Trees {
     override def isType = expansion.isType
   }
 
-  /** A tree representing a quote `'{ expr }
+  /** A tree representing a quote `'{ expr }`
+   *  `Quote`s are created by the `Parser` with an empty `tpt`. In typer
+   *  they can be typed as a `Quote` with a known `tpt` or desugared and
+   *  typed as a quote pattern.
+   *
+   *  `Quotes` are checked transformed in the `staging`, `splicing` and `pickleQuotes`
```

Comment:
Should this be "checked and transformed"? Same in the comment for Splice

Reply:
Yes